public/style.css

```diff
@@ -14,9 +14,13 @@ form.search input[type="text"] {
   float: left;
   width: 80%;
   background: #f1f1f1;
+  border-radius: 10px;
+
 }
 
 form.search button {
+    border-radius: 10px;
```

Comment:
There's still a huge space here and problem with alignment.

Reply:
@jema28 fixed